client/extensions/woocommerce/app/dashboard/required-plugins-install-view.js

```diff
@@ -106,6 +111,7 @@ class RequiredPluginsInstallView extends Component {
 	}
 
 	installPlugins = ( plugins ) => {
+		this.cancelUpdateTimeout();
 		const { site, wporg } = this.props;
 		for ( let i = 0; i < requiredPlugins.length; i++ ) {
```

Comment:
This for loop is getting pretty lengthy and complicated. Might consider simplifying it a bit in the future.